templates/dns.yml.erb

```diff
@@ -5,7 +5,7 @@
 #   <%= "dns_" %>dnscmd (Microsoft Windows native implementation)
```

Comment:
I'm looking at this and wonder if this isn't just ```dns_dnscmd```. Do we need the ruby print thing?